### PR TITLE
Add Qonto Mirage writeup

### PR DESCRIPTION
**Pull request by amyrlam**
## Ember Times review template

- [ ] Add your name to top and bottom
- [ ] Put emoji in writeup title
- [ ] Add a short blurb (could be the title) to the beginning
- [ ] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [ ] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [ ] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-175.md [hunk 1]
```diff
@@ -58,14 +61,11 @@ To learn more how `ember-responsive-image` can help you optimize loading images,
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+[Mirage JS](https://miragejs.com/) is a library for mocking API endpoints as [route handlers](https://miragejs.com/docs/main-concepts/route-handlers/) to test JavaScript apps, without having to rely on any backend services. Sabin discusses how Qonto uses Mirage [Factories](https://miragejs.com/docs/main-concepts/factories/) to test this tricky user flow. Check out the walkthru with test code samples in the [blog post](https://medium.com/qonto-way/how-to-handle-uploading-and-parsing-files-in-your-frontend-app-tests-19bee8e7a61f) to learn about how Qonto mocked the API to make sure it matches what they would expect from the real API in production. Effective testing can pay dividends in improving the developer experience and maintaining the quality of the user experience!
```
Comment (ijlee2) [inline content/the-ember-times-issue-175.md]:
```suggestion
[Mirage JS](https://miragejs.com/) is a library for mocking API endpoints as [route handlers](https://miragejs.com/docs/main-concepts/route-handlers/) to test JavaScript apps, without having to rely on any backend services. Sabin discusses how Qonto uses Mirage [Factories](https://miragejs.com/docs/main-concepts/factories/) to test this tricky user flow. Check out the walkthrough with test code samples in the [blog post](https://medium.com/qonto-way/how-to-handle-uploading-and-parsing-files-in-your-frontend-app-tests-19bee8e7a61f) to learn about how Qonto mocked the API to make sure it matches what they would expect from the real API in production. Effective testing can pay dividends in improving the developer experience and maintaining the quality of the user experience!
```

File: content/the-ember-times-issue-175.md [hunk 2]
```diff
@@ -58,14 +61,11 @@ To learn more how `ember-responsive-image` can help you optimize loading images,
 
 ---
 
-## [4. Section title in sentence case 🐹](section-url)
+## [Blog post: How Qonto used Mirage JS for complex user flow testing ⬆️](https://medium.com/qonto-way/how-to-handle-uploading-and-parsing-files-in-your-frontend-app-tests-19bee8e7a61f)
```
Comment (ijlee2) [inline content/the-ember-times-issue-175.md]: To check, did you mean to include the words blog post in front?

**Review by ijlee2 — APPROVED** [review]
Thank you for the awesome write-up!

**Comment by amyrlam** [comment]
i fixed it, but had some github unicorns so comments never posted. thanks - merging in!

**Pull request merged**